inflation/sdp/InflationSDP.py

```diff
@@ -1751,42 +1766,35 @@ def _update_objective(self) -> None:
                 del self._processed_objective[mon]
         collect()
 
-    def _update_lowerbounds(self) -> None:
-        """Helper function to check that lowerbounds are consistent with the
-        specified known values, and to keep only the lowest lowerbounds
-        in case of redundancy.
-        """
-        for mon, lb in self.moment_lowerbounds.items():
-            self._processed_moment_lowerbounds[mon] = \
-                max(self._processed_moment_lowerbounds.get(mon, -np.infty), lb)
-        for mon, value in self.known_moments.items():
-            if isinstance(value, Real):
-                try:
-                    lb = self._processed_moment_lowerbounds[mon]
-                    assert lb <= value, (f"Value {value} assigned for " +
-                                         f"monomial {mon} contradicts the " +
-                                         f"assigned lower bound of {lb}.")
-                    del self._processed_moment_lowerbounds[mon]
-                except KeyError:
-                    pass
-        self.moment_lowerbounds = self._processed_moment_lowerbounds
-
-    def _update_upperbounds(self) -> None:
-        """Helper function to check that upperbounds are consistent with the
+    def _update_bounds(self, typ: str) -> None:
```

Comment:
Given that this function is only checking consistency, I would change name from `_update_bounds` to `_check_bounds` or something similar.

Reply:
Note that L1795 is modifying `self.moment_xxxerbounds`. I thus believe that the current name is correct.